Modules/_io/_iomodule.c

```diff
@@ -132,9 +132,10 @@ the size of a fixed-size chunk buffer.  When no buffering argument is
 given, the default buffering policy works as follows:
 
 * Binary files are buffered in fixed-size chunks; the size of the buffer
-  is chosen using a heuristic trying to determine the underlying device's
-  "block size" and falling back on `io.DEFAULT_BUFFER_SIZE`.
-  On many systems, the buffer will typically be 4096 or 8192 bytes long.
+  is set to `max(io.DEFAULT_BUFFER_SIZE, st_blksize)` using a heuristic
```

Comment:
same not as `_pyio`, st_blksize is here before it's ever defined. I think don't need to mention it directly.